newrelic/hooks/messagebroker_kombu.py

```diff
@@ -42,13 +42,64 @@
 }
 
 
+def bind_publish(
+    body,
+    routing_key=None,
+    delivery_mode=None,
+    mandatory=False,
+    immediate=False,
+    priority=0,
+    content_type=None,
+    content_encoding=None,
+    serializer=None,
+    headers=None,
+    compression=None,
+    exchange=None,
+    retry=False,
+    retry_policy=None,
+    declare=None,
+    expiration=None,
+    timeout=None,
+    confirm_timeout=None,
+    **properties,
+):
+    return {
+        "body": body,
+        "routing_key": routing_key,
+        "delivery_mode": delivery_mode,
+        "mandatory": mandatory,
+        "immediate": immediate,
+        "priority": priority,
+        "content_type": content_type,
+        "content_encoding": content_encoding,
+        "serializer": serializer,
+        "headers": headers,
+        "compression": compression,
+        "exchange": exchange,
+        "retry": retry,
+        "retry_policy": retry_policy,
+        "declare": declare,
+        "expiration": expiration,
+        "timeout": timeout,
+        "confirm_timeout": confirm_timeout,
+        "properties": properties,
+    }
+
+
 def wrap_Producer_publish(wrapped, instance, args, kwargs):
     transaction = current_transaction()
 
     if transaction is None:
         return wrapped(*args, **kwargs)
 
-    bound_args = bind_args(wrapped, args, kwargs)
+    try:
+        bound_args = bind_publish(*args, **kwargs)
+    except Exception:
+        _logger.debug(
+            "Unable to bind publish arguments. Report this issue to New Relic support.", record_exception=True
```

Comment:
```suggestion
            "Unable to bind arguments for kombu.messaging.Producer.publish. Report this issue to New Relic support.", record_exception=True
```

Make this slightly more specific.